### PR TITLE
IDE-3560 sf on com.liferay.ide.scripting.core

### PR DESCRIPTION
**Pull request by JoyeLuo**

### CODE REVIEW

File: enterprise/plugins/com.liferay.ide.scripting.core/src/com/liferay/ide/scripting/core/GroovyScriptingSupport.java [hunk 1]
```diff
@@ -1,85 +1,70 @@
...
+		}
+		catch (IOException ioe) {
+		}
+
+		return null;
+	}
+
+	public Object newInstanceFromFile(File scriptFile) {
+		@SuppressWarnings("resource")
+		final GroovyClassLoader gcl = new GroovyClassLoader();
```
Comment (jtydhr88) [inline enterprise/plugins/com.liferay.ide.scripting.core/src/com/liferay/ide/scripting/core/GroovyScriptingSupport.java]: should remove this final

**Comment by JoyeLuo** [comment]
got it, will resend

**Comment by jtydhr88** [comment]
I already made the changes and sent to Greg

**Comment by JoyeLuo** [comment]
Thanks!!!

**Pull request closed without merging**